build-system/tasks/csvify-size/index.js

```diff
@@ -174,6 +176,22 @@ function mergeTables(dateTimes, tables) {
 }
 
 /**
+ * @param {string} prettyBytes
+ * @return {number}
+ */
+function reversePrettyBytes(prettyBytes) {
+  var triple = prettyBytes.match(/(\d+(?:\.\d+)?)\ (B|kB|MB|GB|TB|PB|EB|ZB|YB)/);
+  var value = triple[1];
+  var unit = triple[2];
+
+  if (!(value && unit)) {
+    return 0;
+  }
+  var exponent = prettyBytesUnits.indexOf(unit);
+  return (Number(value) * Math.pow(1000, exponent)).toFixed(3);
```

Comment:
i reversing what pretty bytes does https://github.com/sindresorhus/pretty-bytes/blob/master/index.js#L20 so should be 1000 i think